Src/App/ConfigurationManager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace Oat\App;
+
+use Oat\App\Exception\ConfigException;
+
+class ConfigurationManager implements ConfigurationInterface
+{
+    protected $configData;
+
+    /**
+     * ConfigurationManager constructor.
+     * @param $configFilePath
+     * @throws ConfigException
+     */
+    public function __construct($configFilePath)
+    {
+        if (!file_exists($configFilePath)) {
```

Comment:
If file exists but not readable, it will create a fatal error

Reply:
done